src/ItemValue.ts

```diff
@@ -264,10 +264,10 @@ export class ItemValue implements IItemValue {
      * 
      * @returns {Mixed}
      */
-    private retrieveLocalStorage(): void {
-        const value: any = localStorage.getItem(this.ItemsHolder.getPrefix() + this.key);
+    private retrieveLocalStorage(): any {
+        const value: any = this.ItemsHolder.getLocalStorage()[this.ItemsHolder.getPrefix() + this.key];
 
-        if (value === "undefined") {
+        if (!value) {
```

Comment:
Yeah, I saw this as a typo and tried doing `typeof value === "undefined"`, but I got an error and couldn't figure it out. So I changed it to `!value`. Just tried `typeof value === "undefined"` and all tests passed so I'm changing it to that.